warehouse/views.py

```diff
@@ -151,26 +151,6 @@ def forbidden(exc, request):
             )
             return HTTPSeeOther(url)
 
-        # If the forbidden error is because the user doesn't have 2FA enabled, we'll
-        # redirect them to the 2FA page
-        if exc.result.reason in {
-            "owners_require_2fa",
```

Comment:
These are left intentionally because they are required to continue rendering these events in the event history for a project: https://github.com/pypi/warehouse/blob/bcb3fec6d622a3b79116bd085fdd0a50bb37f7db/warehouse/templates/manage/project/history.html#L283-L293

Reply:
Makes sense - I'll leave this comment unresolved for our future selves if we ever want to entertain removing those tags and database events.